cmd/configure/cmd.go

```diff
@@ -88,9 +83,49 @@ func NewCommand() *cobra.Command {
 				output.Errorf("failed to request api7 cloud: %s", err)
 			}
 
+			profileName := options.Global.Configure.Profile
+
+			configuration, err := persistence.LoadConfiguration()
+			if err != nil {
+				output.Verbosef("there is no configuration file, create a new one")
+				configuration = &persistence.CloudConfiguration{}
+				// generate a random profile name if not specified at first time
+				if profileName == "" {
+					profileName = namesgenerator.GetRandomName(0)
+				}
+			} else {
+				if profileName == "" {
+					profileName = configuration.DefaultProfile
+				}
+			}
+
+			newProfile := persistence.Profile{
+				Name:    profileName,
+				Address: options.Global.Configure.Addr,
+				User: persistence.User{
+					AccessToken: options.Global.Configure.AccessToken,
+				},
+			}
+			configuration.ConfigureProfile(newProfile)
+
+			output.Infof("add a new profile: %s", newProfile.Name)
+			if options.Global.Configure.Default || len(configuration.Profiles) == 1 {
+				configuration.DefaultProfile = newProfile.Name
+				output.Infof("profile %s is set as default", configuration.DefaultProfile)
+			}
+
+			if err := persistence.SaveConfiguration(configuration); err != nil {
+				output.Errorf(err.Error())
+			}
+
 			output.Infof("successfully configured api7 cloud access token, your account is %s", me.Email)
 		},
 	}
 
+	cmd.PersistentFlags().StringVar(&options.Global.Configure.Addr, "addr", "https://api.api7.cloud", "Specify the API7 Cloud server address")
+	cmd.PersistentFlags().StringVar(&options.Global.Configure.Profile, "profile", "", "Specify the profile name")
+	cmd.PersistentFlags().BoolVar(&options.Global.Configure.Default, "set-default", true, "Set the profile as default")
```

Comment:
True?

Reply:
Yes, new profile set as default profile by default.

Reply:
I think now let's keep the default value to false.